contributing.md

```diff
@@ -0,0 +1,100 @@
+Contributing
+============
+
+All projects under the Pylons Projects, including this one, follow the
+guidelines established at [How to
+Contribute](http://www.pylonsproject.org/community/how-to-contribute) and
+[Coding Style and
+Standards](http://docs.pylonsproject.org/en/latest/community/codestyle.html).
+
+You can contribute to this project in several ways.
+
+* [File an Issue on GitHub](https://github.com/Pylons/waitress/issues)
+* Fork this project and create a branch with your suggested change. When ready,
+  submit a pull request for consideration. [GitHub
+  Flow](https://guides.github.com/introduction/flow/index.html) describes the
+  workflow process and why it's a good practice.
+* Join the IRC channel [#pyramid](https://webchat.freenode.net/?channels=pyramid) on irc.freenode.net.
+
+
+Git Branches
+------------
+There is a single branch [master](https://github.com/Pylons/waitress/) on which development takes place and from which releases to PyPI are tagged. This is the default branch on GitHub.
+
+
+Running Tests
+-------------
+
+*Note:* This section needs better instructions.
+
+Run `tox` from within your checkout. This will run the tests across all
+supported systems and attempt to build the docs.
+
+To run the tests for Python 2.x only:
+
+    $ tox py2-cover
```

Comment:
Tests should be run for Python 2 and 3, the following will also make sure that full coverage exists:

`tox -e py2-cover,py3-cover,coverage`